pkg/broker/logic.go

```diff
@@ -282,16 +312,16 @@ func (b *BusinessLogic) Update(request *osb.UpdateInstanceRequest, c *broker.Req
 
 func (b *BusinessLogic) LastOperation(request *osb.LastOperationRequest, c *broker.RequestContext) (*broker.LastOperationResponse, error) {
 	response := broker.LastOperationResponse{}
-	
+
 	upgrading, err := b.storage.IsUpgrading(request.InstanceID)
 	if err != nil {
-		glog.Errorf("Unable to get resource (%s) status, IsUpgrading failed: %s\n", request.InstanceID, err.Error()) 
+		glog.Errorf("Unable to get resource (%s) status, IsUpgrading failed: %s\n", request.InstanceID, err.Error())
 		return nil, InternalServerError()
```

Comment:
Can you please turn off auto-formatting on your IDE? It makes pull requests noisy and we agreed we would put global formatting changes into a separate PR.